test/unit/rules/element-attributes-order-test.js

```diff
@@ -0,0 +1,108 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'element-attributes-order',
+
+  config: true,
+
+  good: [
+    '<div @foo="1" bar="baz" {{did-render this.ok}} ...attributes aria-label="foo"></div>',
+    '<div @foo="1" aria-label="foo"></div>',
+    '<div @foo="1" ...attributes></div>',
+    '<div @foo="1" {{did-render this.ok}}></div>',
+    '<div @foo="1" bar="baz"></div>',
+    '<div bar="baz" {{did-render this.ok}} ...attributes aria-label="foo"></div>',
+    '<div bar="baz" aria-label="foo"></div>',
+    '<div bar="baz" ...attributes></div>',
+    '<div bar="baz" {{did-render this.ok}}></div>',
+    '<div {{did-render this.oks}} ...attributes aria-label="foo"></div>',
+    '<div {{did-render this.ok}} ...attributes></div>',
+    '<div ...attributes aria-label="foo"></div>',
+    '<div aria-label="foo"></div>',
+    '<MyComponent @value="5" @change={{this.foo}} local-class="foo" data-test-foo {{on "click" this.foo}} ...attributes as |sth|>content</MyComponent>',
+  ],
+
+  bad: [
+    {
+      template: '<div ...attributes @a="1"></div>',
+      result: {
```

Comment:
Can you please switch all these bad test cases to use:

```js
verifyResults(results) {
  expect(results).toMatchInlineSnapshot(...);
}
```

This is easier to maintain and we updated all our other rules to use this as well.

Info on how to switch: https://github.com/ember-template-lint/ember-template-lint/pull/2150